backend/app/libs/storage_helpers/tokenlists.py

```diff
@@ -19,6 +19,8 @@ async def get_token_list(token_list: str) -> list[Union[str, None]]:
         whitelist = [
             token["address"] for token in resp.json()["tokens"] if token["chainId"] == 1
         ]
+        # ensure native ETH is always whitelisted
+        whitelist.append("0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee")
```

Comment:
Since this is a pretty urgent bug, you could have treated it as a hotfix: make the pull-request and merge it without awaiting review, as soon as the tests pass (self-merge).

Reply:
I just want to be sure I understand the full consequences of this change, I'm really aiming for this PR to not introduce side effects. So I also wanted your perspective of this change as well before merging.

Again, I should mention that with this I _think_ it makes this logic in [`_get_asset_transfer_balances `](https://github.com/butterdao/whip/blob/main/backend/app/treasury/actions.py#L23-L31) and [`_get_token_hist_prices `](https://github.com/butterdao/whip/blob/main/backend/app/treasury/actions.py#L74-84) redundant.

If you can confirm this I'll also remove this logic and add to the PR.

Reply:
From what I understand, `get_treasury` in `backend/app/treasury/adapters/covalent.py` is removing the `ETH` asset because it's not included in the whitelist. Is Covalent sending us the `ETH`/`0xeeee` asset as a part of their `portfolio_v2` response?

If it is indeed the case, we are losing `ETH` as an asset in `Treasury.assets`. This should be fixed ASAP in a different PR.

The solution you are suggesting is fine, although I would have rather placed this line of code in `get_all_token_lists` because `0xee` is not part of any actual tokenlist per se (or even consider move `0xee` alone to a new home-made (and hacky) "ETH fake token" tokenlist).

> Again, I should mention that with this I think it makes this logic in [_get_asset_transfer_balances ](https://github.com/butterdao/whip/blob/main/backend/app/treasury/actions.py#L23-L31)and [_get_token_hist_prices ](https://github.com/butterdao/whip/blob/main/backend/app/treasury/actions.py#L74-84)redundant.

You seem to be right about this, but let's consider this in another PR (separate from this one *and* separate from the hotfix).